rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360AuthorizationServerConfiguration.java

```diff
@@ -95,8 +105,12 @@ public TokenStore tokenStore() {
 
     @Bean
     protected JwtAccessTokenConverter jwtAccessTokenConverter() {
+        Resource resource = new FileSystemResource(new File("/etc/sw360/jwt-keystore.jks"));
```

Comment:
Please remove the hard-coded file path and read it from `CommonUtils.java`

Reply:
Thanks @smrutis1 , I have incorporated the changes.